RecoEgamma/EgammaIsolationAlgos/python/eleIsoFromDeposits_cff.py

```diff
@@ -2,11 +2,11 @@
 
 from RecoEgamma.EgammaIsolationAlgos.eleIsoFromDepsModules_cff import *
 
-eleIsoFromDeposits = cms.Sequence( 
-    eleIsoFromDepsTk * 
-    eleIsoFromDepsEcalFromHitsByCrystal * 
-    eleIsoFromDepsHcalFromTowers *
-    eleIsoFromDepsHcalDepth1FromTowers *
+eleIsoFromDepositsTask = cms.Task( 
+    eleIsoFromDepsTk , 
+    eleIsoFromDepsEcalFromHitsByCrystal , 
+    eleIsoFromDepsHcalFromTowers ,
+    eleIsoFromDepsHcalDepth1FromTowers ,
     eleIsoFromDepsHcalDepth2FromTowers
 )
-
+eleIsoFromDeposits = cms.Task(eleIsoFromDepositsTask)
```

Comment:
This is only used in ```RecoEgamma/EgammaIsolationAlgos/python/eleIsolationSequence_cff.py```: maybe one could avoid defining explicitly the sequence here